deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -122,183 +122,11 @@ spec:
-    ### 1. **For RHEL OS users:** Ensure iSCSI connectivity. If using RHCOS or if the packages are already installed, skip this step and continue to step 2.
-    ### 2. Configure Linux® multipath devices on the host.
-    **Important:** Be sure to configure each worker with storage connectivity according to your storage system instructions. For more information, find your storage system documentation in [IBM Documentation](http://www.ibm.com/docs/).
-
-    **Additional configuration steps for OpenShift® Container Platform users (RHEL and RHCOS).** Other users can continue to step 3.
 
-    Download and save the following yaml file:
-
-    ```bash
-    curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/master/deploy/99-ibm-attach.yaml > 99-ibm-attach.yaml
-    ```
-
-    This file can be used for both Fibre Channel and iSCSI configurations. To support iSCSI, uncomment the last two lines in the file.
-
-    **Important:** The 99-ibm-attach.yaml configuration file overrides any files that already exist on your system. Only use this file if the files mentioned are not already created. <br />If one or more have been created, edit this yaml file, as necessary.
-    Apply the yaml file.
-
-    ```bash
-    oc apply -f 99-ibm-attach.yaml
-    ```
-
-    ### 3. If needed, enable support for volume snapshots (FlashCopy® function) on your Kubernetes cluster.
-
-    For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
-    Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
+    This is the official operator to deploy and manage IBM block storage CSI driver.
 
-    The instructions and relevant yaml files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
-    ### 4.  Configure storage system connectivity.
-    #### 4.1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
-    #### 4.2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
-    ### Installation
-    #### Installing the driver with GitHub
-    The operator for IBM® block storage CSI driver can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
-    Use the following steps to install the operator and driver, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+    For compatibility, prerequisites, release notes, and other user information, see the [IBM block storage CSI driver documentation](https://www.ibm.com/docs/en/stg-block-csi-driver).
```

Comment:
"...information, see [IBM...] (no "the") (per IBM Style)

Reply:
done